front50-gcs/src/main/java/com/netflix/spinnaker/front50/config/GcsConfig.java

```diff
@@ -48,6 +48,9 @@
   @Value("${spinnaker.gcs.bucket}")
   private String bucket;
 
+  @Value("${spinnaker.gcs.bucketLocation}")
```

Comment:
I think you should add an entry to front50.yml (probably pulling the value from spinnaker.yml) and link a comment to
https://cloud.google.com/storage/docs/bucket-locations as a means to document this config parameter.

You dont need to bother with it in default-spinnaker-local.yml .

The default value could be empty.